09.mesh-modeler/02.item-editor/docs.md

```diff
@@ -0,0 +1,90 @@
+
+Let's go through the list of parameters you can set.
+
+
+## Pivots
+
+When you place an item in the map, the mouse cursor is aiming at a particular point in the map. But how to place the item around this point? If your item is a cube, should the editor place the item so that the cursor point is at the center of the cube ? Or the center of the bottom face? Or one corner of the cube?
+
+Pivots are here to answer that question: a pivot is a point of the item that will coincide with the point of the map that is aimed by the cursor.
+
+An item can have multiple pivots. In that case, the map editor might choose automatically one pivot according to the context, or you might be able to press the 'Q' key ('A' with azerty keyboards) to cycle through the pivots of the current item: see the placement parameter *Switch pivot manually* for more information.
+
+If an item has no pivot, the map editor will consider that it has one pivot at the coordinate (0,0,0).
+
+Note: Pivots can also be used as magnets to help align items when you place them on each other: see *Pivot Snap Distance* for more information.
+
+
+## Placement parameters
+
+These are advanced parameters to help the mappers place the item in the map.
+
+They are numerous and sometimes a bit difficult to understand. Just leave the default values, unless you have a specific need for your item.
+
+>>>>>>  Two of them are far more useful than all the others: *Fly Step* and *Grid Horizontal Size*. You can read the explanations of these two and ignore all the others!
+
+
+### Main placement parameters
+
+**Fly Step** If 0, the item will always be placed "on something": the ground, the wall, another item, etc. whatever is aimed by the cursor. If greater than 0, the item can fly above the ground: each time you scroll with the mouse wheel (or press the PgUp/PgDown keys), the item will be offseted vertically by this step (in meters).
+
+**Grid Horizontal Size** If greater than 0, the item will be snapped to the intersections of an invisible horizontal grid, and the distance between the parallel lines of this grid will be this *Grid Horizontal Size* (in meters). If 0, there will be no horizontal snapping, you can place the item exactly where you want (but it will be difficult to align several items or align items with blocks).
+
+
+### Advanced placement parameters
+
+**Switch pivot manually** If activated, the only way to change the pivot to use will be the 'Q' key ('A' with azerty keyboard). If deactivated, the map editor will automatically change the pivot according to the direction of the surface you are aiming with the cursor (for example it may select the lowest pivot if you are aiming at the floor, and the highest pivot if you are aiming at a ceiling). If the editor selects several pivot candidates, you can cycle through them with the 'Q' key.
```

Comment:
```suggestion
**Switch pivot manually** If activated, the only way to change the pivot to use will be the `Q` key (`A` with azerty keyboard). If deactivated, the map editor will automatically change the pivot according to the direction of the surface you are aiming with the cursor (for example it may select the lowest pivot if you are aiming at the floor, and the highest pivot if you are aiming at a ceiling). If the editor selects several pivot candidates, you can cycle through them with the `Q` key.
```